File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/game/ItemUsedMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.game;

public class ItemUsedMessage {

    private String uuid;
    private String type;

    public ItemUsedMessage() {
    }

    public ItemUsedMessage(String uuid, String type) {
        this.uuid = uuid;
        this.type = type;
    }

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }
}

File: client/Client/Network/Messages/Game/ItemUsedMessage.cs
namespace LoadRunnerClient.Network.Messages
{
    public class ItemUsedMessage
    {
        public const string TYPE = "ItemUsedMessage";

        public ItemUsedMessage()
        {
        }

        /// <summary>
        /// ItemUsedMessage Constructor
        /// </summary>
        /// <param name="uuid"></param>
        /// <param name="type"></param>
        public ItemUsedMessage(string uuid, string type)
        {
            this._uuid = uuid;
            this._type = type;
        }

        private string _uuid;

        public string uuid
        {
            get => _uuid;
            set => _uuid = value;
        }

        private string _type;

        public string type
        {
            get => _type;
            set => _type = value;
        }
    }
}